src/test/scala/edu/twitter/classification/ClassifierTest.scala

```diff
@@ -1,12 +1,15 @@
 package edu.twitter.classification
 
+import edu.twitter.config.{AppConfig, DevConfig}
 import edu.twitter.model.impl.gradientboosting.{GradientBoostingBuilder, GradientBoostingModel}
 import edu.twitter.model.service.ModelService
 import org.apache.spark.streaming.{Seconds, StreamingContext}
 import org.apache.spark.{SparkConf, SparkContext}
 
 object ClassifierTest {
   def main(args: Array[String]): Unit = {
+    implicit val appConfig: AppConfig = DevConfig
+
     val conf = new SparkConf().setMaster("local[*]").setAppName("Twitter")
     val sc = new SparkContext(conf)
     val ssc = new StreamingContext(sc, Seconds(10))
```

Comment:
can we put the time of streaming in the AppConfig?

Reply:
Good point :+1: